PackageInfo.g

```diff
@@ -167,6 +251,7 @@ Persons := [
     Email         := "max.horn@uni-siegen.de",
     WWWHome       := "https://www.quendi.de/math",
     PostalAddress := Concatenation(
+                       "Prof. Max Horn\n",
```

Comment:
It’s weird that Max uniquely gets a title ;) But again - I recommend leaving out the names entirely.